src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/servicebus/domains/envelopes/EnvelopeEventProcessor.java

```diff
@@ -191,7 +191,7 @@ private void logMessageParsed(IMessage message, Envelope envelope) {
             envelope.zipFileName,
             envelope.jurisdiction,
             envelope.classification,
-            envelope.caseRef
+            envelope.caseRef == null ? envelope.legacyCaseRef : envelope.caseRef
```

Comment:
and how do we then know if it legacy or not in logs?

Reply:
is this query resolved @doncem ?

Reply:
this query is successfully ignored as very low priority plus initial solution just automagically solves the App Insights report for me/us

Reply:
consider:

```java
envelope.caseRef != null
    ? "Case: " + envelope.caseRef
    : "Legacy case: " + envelope.legacyCaseRef
```
instead of 
```java
envelope.caseRef == null ? "Legacy Case" : "Case",
envelope.caseRef == null ? envelope.legacyCaseRef : envelope.caseRef
```

Reply:
when logging, passing parameters is recommended (and even used to be reported by sonar on failure doing that) instead of mixing up with string concatenation